lhotse/recipes/icsi.py

```diff
@@ -126,9 +127,9 @@
 }
 
 MIC_TO_CHANNELS = {
-    "ihm": [1, 2, 3, 4, 5, 6, 8, 9], # we include 6 since it is used as back-off from some speakers for which no headset-mic exists
-    "sdm": [6],
-    "mdm": ["E", "F", 6, 7],
+    "ihm": ["1", "2", "3", "4", "5", "6", "8", "9"], # we include 6 since it is used as back-off from some speakers for which no headset-mic exists
```

Comment:
what's the reason these are represented as strings now?

Reply:
If not converted to string this line won't work: 

`audio_paths = audio_dir.rglob(f"chan[{channels}].sph")`
https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/icsi.py#L505

Reply:
@LasseWolter thanks, that's exactly the reason. Sorry I was traveling last couple of days so couldn't reply to these comments.